api-admin/src/main/java/bio/terra/pearl/api/admin/service/ScheduledDataRepoExportService.java

```diff
@@ -31,8 +31,7 @@ public void pingDataRepoStatus() {
     if (isTdrConfigured()) {
       log.info("Pinging Terra Data Repo. Up: " + dataRepoExportService.getServiceStatus());
     } else {
-      log.warn(
-          "Error: Skipping TDR status ping, as TDR has not been configured for this environment.");
+      log.info("Skipping TDR status ping, as TDR has not been configured for this environment.");
```

Comment:
Getting rid of "error" in the log message so these don't have to be filtered out of error log queries

Reply:
THANK YOU!